src/main/scripts/src/app/core/workspace.service.ts

```diff
@@ -30,4 +30,10 @@ export class WorkspaceService {
   public organizationCode: string;
   public projectCode: string;
 
+  public isWorkspaceSet(): boolean {
+    this.organizationCode = undefined;
```

Comment:
No need to assign undefined in `is`Function, if orgCode is not set (as seen above) default value is undefined.

Also no need to check length if variable is string shorter version of return should be

```javascript
return this.organizationCode && this.projectCode;
```